app/src/main/res/values-ar/strings.xml

```diff
@@ -445,9 +460,6 @@
     <!-- Text displayed when there are no exceptions, with learn more link that brings users to a tracking protection SUMO page -->
     <string name="exceptions_empty_message_learn_more_link">اطّلع على المزيد</string>
 
-    <!-- Description in Quick Settings that tells user tracking protection is off globally for all sites, and links to Settings to turn it on -->
-    <string name="preferences_tracking_protection_turned_off_globally">مُعطّلة على مستوى المتصفّح، افتح الإعدادات لتفعيلها.</string>
-
```

Comment:
This one was delete here https://github.com/mozilla-mobile/fenix/commit/21cf0b31a456f9d9af1d8b7329ed750bcfaed54f#diff-5e01f7d37a66e4ca03deefc205d8e7008661cdd0284a05aaba1858e6b7bf9103L436, still checking if it's safe